docs/stability.md

```diff
@@ -0,0 +1,25 @@
+# Stability of the AWS SDK for Swift
+
+For information about maintenance and support of SDK major versions and their underlying dependencies, see the
+following in the AWS SDKs and Tools Shared Configuration and Credentials Reference Guide:
+
+* [AWS SDKs and Tools Maintenance Policy](https://docs.aws.amazon.com/credref/latest/refdocs/maint-policy.html)
+* [AWS SDKs and Tools Version Support Matrix](https://docs.aws.amazon.com/credref/latest/refdocs/version-support-matrix.html)
+
+## Qualifiers
+
+Qualifiers for published artifacts
+
+**`-alpha`** indicates:
+
+* The SDK is not meant for production workloads. It is released solely for the purpose of getting feedback.
+* The SDK is not yet feature complete. It may contain bugs and performance issues.
+* Expect migration issues as APIs/types change.
+
+**`-beta`** indicates:
+
+* The SDK is not meant for production workloads. 
+* The SDK is feature complete. It may still contain bugs and performance issues.
+* The APIs/types are mostly stabilized. It is still possible that future releases may cause migration issues.
```

Comment:
We don't have preview labels (`-alpha`, `-beta`) in the versions. Like Kotlin SDK https://mvnrepository.com/artifact/aws.sdk.kotlin/aws-core-jvm/0.17.6-beta

This section must not exist for Swift.

That doesn't mean we can't to prerelease identitfier. SPM has support for it already https://github.com/apple/swift-package-manager/blob/main/Sources/PackageDescription/Version.swift#L50

But as of now, let's just remove it and create a GitHub issue to track this.